lib/broccoli/fastboot-config.js

```diff
@@ -104,7 +104,9 @@ FastBootConfig.prototype.readAssetManifest = function() {
     var assetMap = JSON.parse(fs.readFileSync(assetMapPath));
     return assetMap;
   } catch (e) {
-    // No asset map was found, proceed as usual
+    if (assetMapPath) {
```

Comment:
I thought about that. Any time getting an `assetMap` load fails we should log it. If it's a malformed `assetMap` that fails `parse` or if it's just absent, the user would benefit from knowing that.  We can print the error too.